### ISSUE TITLE
Polski? W JAVIE?

### ISSUE THREAD

**Issue by Mcpg** [grzegorz2047/AceOfSpadesMiniGame#3]
Czy ty w ogóle znasz "zasady nazewnictwa" w Javie wg. Oracle?!
NIE PISZE SIĘ NUPIE PO POLSKU (nazw, tekst w stringach już po polsku może być.)!!!!!!!!!!!!!!!!!!!!!1111ONEONEONONEENIEREINEREINER
Wiem, że jesteśmy Polakami etc, ale takie błędy mnie drażnią...

Poza tym to formatowanie...
"public static void main(String[] args){" - nie da się nic odczytać...

Uff. Miło było mi się trochę wyżyć... :+1:

**Comment by artur9010** [comment]
Wykastrować dziada!

**Comment by grzegorz2047** [comment]
@MCPlayG Tak. Formatowanie jest takie, jakie mi odpowiada. Jest to jedno z dwóch popularnych formatowań. Java nie ma z tym problemów btw. zobacz tutaj jakie jest formatowanie: http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141270.html. Co do języka polskiego to wiadomo, lepiej pisać po angielsku. W tamtym czasie pisałem różnie i doszedłem do wniosku, że polski się nie nadaje do programowania. Znam zasady !!!!1!!!!!!One1one!!!!!! A jak coś nie pasuje to sobie popraw xd Droga wolna.
